lfs/scanner_git_test.go

```diff
@@ -0,0 +1,76 @@
+package lfs_test // to avoid import cycles
+
+// This is for doing complete git-level tests using test utils
+// Needs to be a separate file from scanner_test so that we can use a diff package
+// which avoids import cycles with testutils
+
+import (
+	"testing"
+
+	. "github.com/github/git-lfs/lfs"
```

Comment:
Yeah the general advice is to not use `import . foo`, but there are a couple of officially sanctioned exceptions in test code - it's mostly just so it feels like normal test code even though it has to be in a different package to break cycles.